log/loggers.go

```diff
@@ -5,122 +5,244 @@ import (
 	"log"
 )
 
-// Info takes a pointer subLogger struct and string sends to newLogEvent
+// Info takes a pointer subLogger struct and string sends to StageLogEvent
 func Info(sl *SubLogger, data string) {
+	mu.RLock()
+	defer mu.RUnlock()
 	fields := sl.getFields()
-	if fields == nil || !fields.info {
+	if fields == nil {
 		return
 	}
-
-	displayError(fields.logger.newLogEvent(data,
-		fields.logger.InfoHeader,
-		fields.name,
-		fields.output))
+	if fields.info {
+		fields.output.StageLogEvent(func() string { return data },
+			fields.logger.InfoHeader,
+			fields.name,
+			fields.logger.Spacer,
+			fields.logger.TimestampFormat,
+			fields.logger.ShowLogSystemName,
+			fields.logger.BypassJobChannelFilledWarning)
+	}
+	logFieldsPool.Put(fields)
 }
 
-// Infoln takes a pointer subLogger struct and interface sends to newLogEvent
+// Infoln takes a pointer subLogger struct and interface sends to StageLogEvent
 func Infoln(sl *SubLogger, v ...interface{}) {
+	mu.RLock()
+	defer mu.RUnlock()
 	fields := sl.getFields()
-	if fields == nil || !fields.info {
+	if fields == nil {
 		return
 	}
-	displayError(fields.logger.newLogEvent(fmt.Sprintln(v...),
-		fields.logger.InfoHeader,
-		fields.name,
-		fields.output))
+	if fields.info {
+		fields.output.StageLogEvent(func() string { return fmt.Sprintln(v...) },
+			fields.logger.InfoHeader,
+			fields.name,
+			fields.logger.Spacer,
+			fields.logger.TimestampFormat,
+			fields.logger.ShowLogSystemName,
+			fields.logger.BypassJobChannelFilledWarning)
+	}
+	logFieldsPool.Put(fields)
 }
 
-// Infof takes a pointer subLogger struct, string & interface formats and sends to Info()
+// Infof takes a pointer subLogger struct, string & interface formats
```

Comment:
inconsistent comment style
```suggestion
// Infof takes a pointer subLogger struct, string and interface formats sends to StageLogEvent
```